src/microscope_tiling_basler_camera/pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[project]
+name = "microscope_tiling_basler_camera"
+version = "0.1.0"
+description = "Default template for PDM package"
+authors = [
+    {name = "Colby Sparks", email = "colby.sparks@fwee-colby.izaber.com"},
+]
+dependencies = [
+    "numpy>=1.26.4",
+    "pypylon>=3.0.1",
+    "opencv-python>=4.9.0.80",
+    "zaber-motion>=5.2.0",
+    "imutils>=0.5.4",
+]
```

Comment:
I'd change all the dependencies to a fixed or compatible version. I don't know why the default is min-version.

Reply:
fixed